velox/docs/develop/types.rst

```diff
@@ -73,6 +73,27 @@ Examples:
 * Timestamp(-5000*24*60*60 - 1000, 123456) represents 1956-04-24 07:43:20.000123456
   (5000 days 1000 seconds before epoch plus 123456 nanoseconds).
 
+Floating point types (REAL, DOUBLE) have special values negative/positive infinity and
+not-a-number (NaN).
+
+For NaN the semantics are different than the standard floating point semantics:
```

Comment:
NaN semantics are different from C++.

(it is not clear which standard we are talking about? C++, SQL, or something else)